tests/test_analysis.py

```diff
@@ -17,7 +17,51 @@ def test_simplecase(self):
         expected_profit = (expected_income * (-1)) - expected_expenses
 
         # When
-        actual_profit = total_profit(balances)
+        actual_profit = data_analysis.total_profit(balances)
 
         # Then
         self.assertEqual(expected_profit, actual_profit)
+
+
+class TestCumulativeProfit(unittest.TestCase):
+
+    def test_simplecase(self):
```

Comment:
Do we want more descriptive test names? One convention is test_function_situation()

Reply:
done

Reply:
Did you push?

Reply:
done - changed the names of the other ones as part of the other PR so only this test case (for cumulative profit) will be updated on this branch